### ISSUE TITLE
Failed to load plugin 'jsx-a11y'

### ISSUE THREAD

**Issue by pnvskp** [jsx-eslint/eslint-plugin-jsx-a11y#758]
I am getting this error at the starting of every file.

Error: Failed to load plugin 'jsx-a11y' declared in 'package.json >> eslint-config-react-app':Cannot read property 'href' of undefined.

Can anyone help me in resolving this

**Comment by ljharb** [comment]
Do you have this plugin installed? Is there any more info on the error?

**Comment by NikkiJonesR** [comment]
`cannot read property 'href' of undefined tracing to standardBrowserEnv - node_modules/axe-core/axe.js:4098:8`

**Comment by ljharb** [comment]
What version of eslint-plugin-jsx-a11y and axe-core do you have? (If not the latest, please update them and re-check?)

**Comment by NikkiJonesR** [comment]
We're using the latest version `^6.4.1`

**Comment by ljharb** [comment]
and axe-core?

**Comment by pnvskp** [comment]
> Do you have this plugin installed? Is there any more info on the error?

Yeah, it is present in node_modules folder.

I didn't face this error until now. 
It popped up right after opening the app in Atom after I created it using npx create-react-app.

I have no other info regarding this.

**Comment by ljharb** [comment]
Does it happen on the command line? If not, it’s either an atom bug, or you need to configure it to only use a local eslint (you shouldn’t even have a global one installed)

**Comment by NikkiJonesR** [comment]
> Does it happen on the command line? If not, it’s either an atom bug, or you need to configure it to only use a local eslint (you shouldn’t even have a global one installed)

Confirming this is an Atom specific bug, it doesn't occur on VSCode :(
I don't have eslint globally installed.

**Comment by dprentis** [comment]
Upgrading axe-core to version 4.1.1 fixed this issue for me

**Comment by pnvskp** [comment]
> Upgrading axe-core to version 4.1.1 fixed this issue for me

Ok, will look into it. 
Thanks for the info.
For now, I have uninstalled eslint globally the issue stopped popping up.
